.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  # Manual invocation.
+  workflow_dispatch:
+
+  push:
```

should this run only on pushes to `main`?

```suggestion
  push:
    branches:
      - main
```

---

nope, this is just the build (rather than deploy) so if we want to be able to deploy to CODE to test, we don't want to have to manually run the action, there is an argument for doing it on `pull_request` but I personally find that annoying too. This is low activity repo, so I don't think we need to worry about build every branch.

---

Deployment of this repository puts files into a publicly accessible bucket, whose objects are cached quite aggressively. We don't (yet) have a process of deleting files from S3 during deployment.

I quite like the idea of restricting builds to pushes to main, as it means builds are peer-reviewed, and the hope is the chance of uploading the wrong files to S3 is reduced. That is, we don't accidentally make a private use font globally public (assuming that's a valid concern!).